packages/block-library/src/list-item/hooks/use-space.js

```diff
@@ -34,12 +36,30 @@ export default function useSpace( clientId ) {
 					return;
 				}
 
+				if ( keyCode !== SPACE && keyCode !== TAB ) {
```

Comment:
How do you see this working when tab moves to the block sidebar?

Reply:
You'd have to move focus back to the list. Not saying this is an ideal solution. I think it might be better to show a tooltip when you're in the right spot to use Space and also announce it for screen readers.

Reply:
@ellatrix I would have to agree unless you changed the message to also give those instructions. Focus the list, navigate to the item you wish to indent, use space instead of tab.